addons/udes_stock_packaging/data/package_types.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<odoo>
+  <data>
+    <!--Pallet package type-->
+    <record id="sequence_pallet" model="ir.sequence">
+      <field name="name">Package Type Pallet</field>
+      <field name="prefix">UDES</field>
+      <field name="padding" eval="5"/>
+      <field name="code">stock.quant.package.pallet</field>
+    </record>
+
+    <record id="pallet_package_type" model="package.type">
+      <field name="name">Pallet</field>
+      <field name="sequence">1</field>
+      <field name="sequence_id" ref="sequence_pallet"/>
+      <field name="package_type_regex">^UDES(?:[0-9])+$</field>
```

Comment:
Nit: I'm not sure we need the non-capturing group in these regexes? `^UDES[0-9]+$` should be fine I think.

Reply:
I believe copied it from pallet regex on the warehouse which we have been using, but yeah i agree with you